openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -501,6 +504,41 @@ const TeamDetailsV1 = ({
     updateTeamHandler(updatedTeamData);
   };
 
+  const handleReactiveTeam = async () => {
+    const teamChildren = generateIdArray(childTeams as EntityReference[]);
+    const updatedUserData: CreateTeam = {
+      teamType: currentTeam.teamType as TeamType,
+      name: currentTeam.name,
+      isJoinable: currentTeam.isJoinable,
+      defaultRoles: generateIdArray(currentTeam.defaultRoles),
+      children: isEmpty(teamChildren) ? undefined : teamChildren,
+      description: currentTeam.description,
+      displayName: currentTeam.displayName,
+      parents: generateIdArray(currentTeam.parents),
+      owner: currentTeam.owner,
+      policies: generateIdArray(currentTeam.policies),
+      profile: currentTeam.profile,
+      users: generateIdArray(currentTeam.users),
+    };
```

Comment:
Why we are not using cloneDeep here as I suggested earlier?

Reply:
@chirag-madlani Updated!